pkg/kudoctl/cmd/init_integration_test.go

```diff
@@ -1,5 +1,3 @@
-// +build integration
-
 package cmd
```

Comment:
is there a reason to remove `+build integration` tag?   Should the class be renamed to reflect it?

Reply:
no, thank you